action.yml

```
@@ -17,6 +17,9 @@ inputs:
     default: ${{ github.token }}
   cache-dependency-path:
     description: 'Used to specify the path to dependency files. Supports wildcards or a list of file names for caching multiple dependencies.'
+  no-environment-update:
+    description: 'Set this option if you want the action not to update environment variables.'
+    default: false
```

```suggestion
  update-environment:
    description: 'Set this option if you want the action to update environment variables.'
    default: true
```

Putting boolean inputs in positive terms is usually easier for people to understand.

---

done